ruby/ql/lib/codeql/ruby/controlflow/CfgNodes.qll

```diff
@@ -751,4 +751,23 @@ module ExprNodes {
     /** Gets a pair of this hash literal. */
     PairCfgNode getAKeyValuePair() { result = this.getAnArgument() }
   }
+
+  private class CallableExprChildMapping extends ExprChildMapping, Callable {
+    // TODO: parameters?, callable body?
+    override predicate relevantChild(AstNode n) { none() }
+  }
+
+  /** A control-flow node that wraps a `Callable` AST expression. */
+  class CallableCfgNode extends ExprCfgNode {
```

Comment:
I think these classes are not very useful. The control flow of a Callable is just a single node representing the creation of the function-like object. There is no flow to the body or its parameters.